libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -458,6 +474,36 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         }
         return;
     }
+
+    /**
+     * @hidden
+     * if `columnLayoutType` is given, set those column layouts appropriately
+     */
+    private _setUserSpecifiedLayout(groupField: GroupField): void {
+        const layoutString = this.columnLayoutType.split('-');
```

Comment:
You can use destructuring assignment, `const [x,l,m]= this.columnLayoutType.split('-');`
and more readable
`parseInt(x.slice(2, x.length), 10);`

Reply:
fixed as suggested, thanks for the comment!